Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -4187,6 +4187,7 @@ def gen2023HiMix(fragment,howMuch):
 defaultDataSets['2026D95']='CMSSW_13_1_0_pre1-130X_mcRun4_realistic_v2_2026D95noPU-v'
 defaultDataSets['2026D96']='CMSSW_13_1_0_pre1-130X_mcRun4_realistic_v2_2026D96noPU-v'
 defaultDataSets['2026D98']='CMSSW_13_2_0_pre1-131X_mcRun4_realistic_v5_2026D98noPU-v'
+defaultDataSets['2026D110']='CMSSW_14_1_0_pre2-140X_mcRun4_realistic_v3_STD_2026D110_noPU-v'
```

Comment:
Would you mind to add the comment on GT here, so we don't question in future if GT is wrong.

Reply:
Sure. Maybe I was thinking to directly have a new production with the proper GT string.